web/libs/ui/package.json

```
@@ -3,7 +3,9 @@
   "version": "0.0.0",
   "license": "MIT",
   "private": true,
-  "dependencies": {},
+  "dependencies": {
+    "@radix-ui/react-select": "^2.1.6"
+  },
```

```suggestion
  "dependencies": {},
```

This shouldn't have its own dependencies as it is a monorepo